pyproject.toml

```diff
@@ -20,15 +20,15 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
 numpy = "^1.20.0"
-numba = "^0.55"
-pystencils = "^1.0"
+numba = ">=0.55"
 scipy = "^1.7.3"
 sympy = "^1.9"
-pyfftw = "^0.13.0"
 psutil = "^5.9.0"
 h5py = "^3.6"
 pyelastica = "^0.3"
-matplotlib = "^3.3.2"
+matplotlib = "3.7.1"
+pyfftw = { git="https://github.com/tp5uiuc/pyFFTW.git", branch="macos_fix" }
+pystencils = { git="https://github.com/sy-cui/pystencils.git", branch="macos_fix" }
```

Comment:
Is this repo private or dented I can't access?

Reply:
Made the repo public. Should work now